troubleshooting/app-does-not-connect-to-remote-point.md

```diff
@@ -28,7 +29,7 @@ When running an app on a device, its requests to a remote point via HTTPS do not
 The issue is observed on iOS devices with any build configuration and on Android devices when the app is built with **Release** configuration.
```

Comment:
Perhaps it is more clear to state: on iOS devices with either **Debug** or **Release** build configuration and on Android.....